microsetta_interface/implementation.py

```diff
@@ -2182,6 +2235,16 @@ def get_consent_view(*, account_id=None, source_id=None, consent_type=None):
         False
     )
 
+    consent_type_parts = consent_output['consent_type'].split("_")
+    if consent_type_parts[0] == "adult":
+        age_range = "18-plus"
+    elif consent_type_parts[0] == "adolescent":
+        age_range = "13-17"
+    elif consent_type_parts[0] == "child":
+        age_range = "7-12"
+    else:
+        age_range = "0-6"
```

Comment:
not sure if this is the correct match but essentially let's be defensive and exact

```suggestion
    elif consent_type_lifestage == 'infant':
        age_range = "0-6"
    else:
        raise ValueError(f"This shouldn't happen: {consent_type_lifestage}")
```